### PR TITLE
PP-5170: move backgroundProcessingEnabled configuration entry to CaptureProcessConfig

### PR DESCRIPTION
**Pull request by jankowiakmaria**
## WHAT YOU DID
move backgroundProcessingEnabled configuration entry to CaptureProcessConfig (https://github.com/alphagov/pay-connector/pull/1330#pullrequestreview-243139070)

### Documentation
- Updated README.md for new environment variables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,18 +38,19 @@ The GOV.UK Pay Connector in Java (Dropwizard)
...
 The polling interval is a random value between 150-200 seconds.
 
 If a capture attempt fails it will be retried again after a specified delay (`CAPTURE_PROCESS_RETRY_FAILURES_EVERY`).
 
-The following variables control the background process: 
+The following variables control the background process:
 
 | Varible | Default | Purpose |
 |---------|---------|---------|
+| `BACKGROUND_PROCESSING_ENABLED` | `true` | enables registering scheduled processes - it includes both new and old capture methods |
```
Comment (kbottla) [inline README.md]: May be we can specify `Database based & queue based capture methods` instead of `new and old` ?

**Review by kbottla — COMMENTED** [review]
Looks good. Just a minor comment

File: README.md (hunk 1 above)
Comment (jankowiakmaria) [reply README.md]: Updated - tests re-running. Feel free to approve though.

**Review by kbottla — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (kbottla) [reply README.md]: 👍

**Comment by jankowiakmaria** [comment]
Snyk is hanging.
Passed previous build.
Merging.

**Pull request merged**